compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.td

```diff
@@ -212,6 +220,19 @@ def DumpDispatchGraphPass : Pass<"iree-flow-dump-dispatch-graph-pass"> {
   ];
 }
 
+def ElementwiseOpFusionPass :
+    Pass<"iree-flow-elementwise-op-fusion", ""> {
+  let summary = "Fuse elementwise operations";
+  let options = [
+    Option<"fuseMultiReduction", "fuse-multi-reduction", "bool",
+           /*default=*/"true", "Fuse ops that have multiple reduction iterators">
+  ];
+  let dependentDialects = [
+    "mlir::affine::AffineDialect",
+  ];
+}
+
+
 def ExportBenchmarkFuncsPass :
     Pass<"iree-flow-export-benchmark-funcs-pass", "mlir::ModuleOp"> {
   let summary = "Exports benchmark functions";
```

Comment:
Can you add a description to `FusionOfTensorOpsPass`? After separating elementwise fusion and fusion by expansion that pass seems to mostly be focused on multi-use fusion and is no longer reflected by the name of the pass. Or change the name of the pass maybe. I noticed that a number of Flow passes could use better names (in particular the many dimension collapsing passes).

Reply:
There are a few things to unwind with that pass still. I need to
1) Remove constant folding patterns there
2) Remove the collapse shape propagation patterns from there as well.

Once I get through 1 and 2, ill rename the `FusionOnTensors` pass.